src/Parser/TokenIterator.php

```diff
@@ -59,7 +60,7 @@ public function currentTokenOffset(): int
 
 	public function isCurrentTokenValue(string $tokenValue): bool
 	{
-		return $this->tokens[$this->index][Lexer::VALUE_OFFSET] === $tokenValue;
+		return strcasecmp($this->tokens[$this->index][Lexer::VALUE_OFFSET], $tokenValue) === 0;
```

Comment:
What is this change for?

Reply:
This supports e.g. `T IS NOT NULL`. If you prefer only to support lowercase (i.e. `T is not null`) I'll revert this

Reply:
Yes, please revert, I'm fine with supporting only a single variant :)